styles/styles.css

```diff
@@ -0,0 +1,1844 @@
+    margin-top: 0;
+    padding: 0;
+    font-size: 1.2rem;
+  }
+  .blog-text {
+    color: #000000;
+    width: 80%;
+    display: none;
+  }
+  .blog-image {
+      width: 50%;
+  }
+  .dog-title-travel {
+    justify-content: space-around;
+    align-items: center;
+    color: #696969;
+    margin-left: 0;
+    font-family:Verdana, Geneva, Tahoma, sans-serif;
+    font-style: normal;
+  }
+  fieldset {
+    width: 20%;
+    justify-content: left;
+    padding: 0;
+    margin: 0;
+  }
+  .dogs-adopted {
+    color:#2f4f4f;
+    font-size: 1.5rem;
+    font-weight: bold;
+    padding-bottom: 1rem;
+  }
+  .first-time {
+    display: flex;
+    flex-direction: row;
+    padding: 0.5rem;
+    align-self: flex-start;
+  }
+  .adopt-final{
+    width: 10%;
+  }
+  .adopted-dogs {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+  }
+  .adopted-dogs img {
+    width: 50%;
+    align-items: center;
+  }
+  .adopted-text {
+    flex-direction: column;
+    line-height: 0.8rem;
+    width: 100%;
+    font-size: .5rem;
+    font-family:'Times New Roman', Times, serif;
+    color: #008080;
+  }
+  .inline-price {
+    display: flex;
+    flex-direction: column;
+    font-size: 1rem;
+    width: 50%;
+    color: #000000;
+  }
+  .total-cost {
+    color: #000000;
+    font-weight: 500;
+    font-family:Verdana, Geneva, Tahoma, sans-serif;
+    font-size: 1.5rem;
+  }
+  .total-price {
+    font-size: 1.5rem;
+    color: #000000;
+    padding-left: 1rem;
+  }
+  .price-adopt {
+    color: #000000;
+    font-weight: 100;
+    margin-left: 1rem;
+    font-size: 1rem;
+  }
+  .inline-price {
+    display: flex;
+    flex-direction: row;
+    font-size: 1rem;
+    width: 100%;
+    color: #000000;
+  }
+  .going-home {
+    color:#000000;
+  }
+  .see-more {
+    display: flex;
+    width: 15rem;
+    justify-content: center;
+    text-align: center;
+    align-items: center;
+    font-weight: bold;
+    text-decoration: none;
+    font-family:Verdana, Geneva, Tahoma, sans-serif;
+    font-size: 1rem;
+  }
+  .last-button {
+    width: 25%;
+    height: 3rem;
+    font-size: 1.2rem;
+    font-family:Verdana, Geneva, Tahoma, sans-serif;
+    font-weight: bold;
+    background-color: #88cc00;
+    color: white;
+    margin: 0 0 2rem 1rem;
+    border: 0;
+  }
+  /*------------FOOTER--------*/
+  footer {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    align-items: center;
+    flex-wrap: wrap;
+    background-color: #003366;
+    color: #ffffff;
+  }
+  footer p {
+    text-align: center;
+    font-size: 1rem;
+    line-height: 1rem;
+  }
+  .footer-logo {
+    max-height: 5rem;
+    font-weight: bold;
+  }
+  .nav-footer {
+    display: flex;
+    flex-direction: row;
+  }
+  .nav-item-ft {
+    color: #ffffff;
+    text-decoration: none;
+  }
+}
+/*XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX*/
+/*--Media Screen 768px tablet Phone--*/
+/*XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX*/
+@media only screen and (min-width: 768px) {
```

Comment:
Hi Suzanne. Sorry to hear you're having a hard time with the grid. While you're on the right track here, I'd like you to check out this article on creating a responsive grid and see if you can replicate what they're doing. This is probably the easiest way of making your grid responsive. In your media queries, all you'll have to set is the amount of columns. See if this helps: https://travishorn.com/responsive-grid-in-2-minutes-with-css-grid-layout-4842a41420fe